css/styles.css

```diff
@@ -0,0 +1,78 @@
+/*
+- Font sizes (px):
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+
+- Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+*/
+
+:root {
+  font-size: 10px;
+  font: 400 10px / 1.2 "Yeseva One";
+}
+
+.container {
+  max-width: 160rem;
+  margin: 0 auto;
+}
+
+.header {
+  font-size: 2rem;
+}
```

Comment:
Looks like there're different font sizes in the header, not only `2rem`. Therefore font size is expected to be set as close as possible. Avoid property overlap.